integration/testdata/conan.json.golden

```diff
@@ -26,7 +26,7 @@
           "Name": "poco",
           "Identifier": {
             "PURL": "pkg:conan/poco@1.9.4",
-            "UID": "312753cebe80c0eb"
+            "UID": "1c5e3b385e0d9c68"
```

Comment:
Do you have any idea why UID is different now?

Reply:
I added `workspaceRelationshi`p as 2:
https://github.com/aquasecurity/trivy/blob/542a3808239e08a41087f8af531efe8762958f29/pkg/fanal/types/package.go#L17-L21
to correct sorting:
https://github.com/aquasecurity/trivy/blob/927c6e0c9d4d4a3f1be00f0f661c1d18325d9440/pkg/fanal/types/package.go#L233-L240

Reply:
It makes sense, thanks!